variables.tf

```diff
@@ -293,3 +293,9 @@ variable "create_cpr" {
   type        = bool
   default     = false
 }
+
+variable "automount_service_account_token" {
+  description = "Instructs the cluster whether or not to automatically mount a kubernetes service account token.  Past GKE 1.24, tokens are not automatically created for a service account"
```

Comment:
tokens are still auto-created and mounted in in 1.24, they are just ephemeral rather than stored in `Secret` API objects